From 6457265356586871249a2a946dc575662cbb6b76 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Tue, 18 Jun 2024 09:37:00 +0530
Subject: [PATCH] Update libdl bionic library.

Snapuserd module is dependent on libdl bionic library. Since snapuserd
should be apex platform avaiable, then dependent libdl library should
also be make apex platform avaiable.

Tests: Build the EB, there was no issue in these two modules.

Tracked-On: NA
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 libdl/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libdl/Android.bp b/libdl/Android.bp
index 95b412b96..533d39f29 100644
--- a/libdl/Android.bp
+++ b/libdl/Android.bp
@@ -48,6 +48,7 @@ cc_library_static {
         never: true,
     },
     apex_available: [
+        "//apex_available:platform",
         "com.android.runtime",
     ],
 }
-- 
2.34.1

